src/site/content/en/blog/optimize-lcp/index.md

```diff
@@ -38,29 +38,90 @@ To provide a good user experience, **sites should strive to have an LCP of 2.5 s
-  {% YouTube "fWoI9DXmpdk" %}
+Where CrUX does not provide data (for example, a page with insufficient traffic to get page-level data), CrUX should be supplemented with RUM data collected using JavaScript APIs running on the web page. This can also provide much more data than CrUX can expose as a public dataset. Later in this guide we will explain how to collect this data using JavaScript.
 {% endAside %}
 
+PageSpeed Insights shows up to four different CrUX data:
+- **Mobile** data for **This URL**
+- **Desktop** data for **This URL**
+- **Mobile** data for the whole **Origin**
+- **Desktop** data for the whole **Origin**
+
+These can be toggled in the controls at the top, and top right-hand side of this section. Be aware that where a URL does not have sufficient data to be shown at URL-level, but does have data for the origin, PageSpeed Insights will automatically show this.
+
+{% Img src="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/wWQxf1DbjElItJTcJsBn.png", alt="PageSpeed Insight's falling back to origin-level data where url-level data is not available", width="800", height="295" %}
+
+The LCP for the whole origin may be very different to an individual page's LCP depending on how the LCP is loaded on that page compared to other pages on that origin. It can also be affected by how visitors navigate to these pages. Home pages tend to be visted by new users and so may often be loaded "cold", without any cached content and so are often the slowest pages on a website.
+
+Looking at the four difference pieces of CrUX data can help you understand whether an LCP issue is specific to this page, or a more general site-wide issue. Similarly, it can show which device types have LCP issues.
+
+### Using PageSpeed Insights CrUX supplementary metrics
+
+As well as LCP, those looking to optimize LCP, should also use the [First Contentful Paint (FCP)](/fcp/) and [Time to First Byte (TTFB)](/ttfb/) timings which can provide valuable insights into LCP.
+
+TTFB is the time when the visitor starting to navigate to a page (for example, clicking on a link), until the first bytes of the HTML document are received. A high TTFB can make achieving a 2.5 second LCP challenging, or even impossible.
+
+A high TTFB can be due to mutiple server redirects, visitors located far away from the nearest site server, visitors on poor network conditions, or an inability to use cached content due to query parameters.
+
+Once a page starts rendering there may be an initial paint (for example, the background color is painted), followed by some content showing up (for example, the site header). The appearnce of the initial conent is measured by FCP. The delta between FCP and other metrics can be very telling.
+
+A large delta between TTFB and FCP could indicate that the browser needs to download a lot of render-blocking assets. It can also be a sign it must complete a lot of work to render any meaningful content—a classic sign of a Client Side Rendered site.
```

Comment:
```suggestion
A large delta between TTFB and FCP could indicate that the browser needs to download a lot of render-blocking assets. It can also be a sign it must complete a lot of work to render any meaningful content—a classic sign of a site that relies heavily on client-side rendering.
```